internal/api/http.go

```diff
@@ -187,8 +187,17 @@ func inspectableMIMEType(t string) bool {
 	return strings.HasPrefix(t, "text/") || jsonTypeRE.MatchString(t)
 }
 
-func isValidHost(reqHost, hrtHost string) bool {
-	return strings.Contains(reqHost, hrtHost)
+func isRequestHostValid(reqHost, hrtHost string) bool {
```

Comment:
Since this function checks whether the request host is under a specific domain (e.g. `github.com`), and not whether a hostname is "valid" (which is vague and unspecified), how about renaming it to `isSameDomain()`?

Reply:
Good idea - I've renamed as suggested